src/libostree/ostree-repo.c

```diff
@@ -2094,6 +2094,27 @@ append_remotes_d (OstreeRepo          *self,
 }
 
 gboolean
+ostree_repo_is_created (OstreeRepo    *self,
+                        gboolean      *does_exist,
+                        GError        **error)
+{
+  struct stat stbuf;
+
+  if (lstat (gs_file_get_path_cached (self->objects_dir), &stbuf) < 0)
```

Comment:
Hmm...we could change them to share more code over time, but as is now it's only ~10 lines in this function, I'm okay with it